CapstoneProject/ViewControllers/ActivityRecViewController.m

```diff
@@ -5,17 +5,16 @@
 //  Created by Apoorva Chilukuri on 7/13/22.
 //
 
-#import "ActivityRecViewController.h"
-#import <Parse/Parse.h>
-
 #import "ActivityRecViewController.h"
 #import "ActivityTableViewCell.h"
 #import "ActivityRecommendation.h"
-
+#import "MoodDetection.h"
+#import <Parse/Parse.h>
 
 @interface ActivityRecViewController ()
 @property (weak, nonatomic) IBOutlet UITableView *tableView;
 - (IBAction)didTapChooseCustomeAct:(id)sender;
+@property (weak, nonatomic) IBOutlet UILabel *activityRecommendationText;
```

Comment:
I believe this label is inside the table view cell, so you need to create a class for your cell add the outlet to the cell class rather than to view controller class.